<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="set_params" default="true"/>
    <arg name="log_name" default="log"/>
    <node name="script_loadGoal" pkg="platinum_bridge" type="load_goals.sh"/>
    <node name="mongodb_goals" pkg="platinum_bridge" type="getGoals.py" output="screen"/>
    <node name="platinum_bridge" pkg="platinum_bridge" type="bridge" args="$(arg set_params) $(arg log_name)" output="screen" launch-prefix="bash -c 'sleep 1; $0 $@'"/>
    <!-- <node name="platinum_bridge" pkg="platinum_bridge" type="bridge" output="screen"/> -->
</launch>
